tests/load_interrupt/run.sh

```diff
@@ -33,12 +33,43 @@ function check_row_count() {
 	[ "$estimate" == "$row_count" ]
 }
 
-function run() {
-	THRESHOLD=1024
+function test_save_checkpoint_faild() {
 	prepare_datafile
-
 	run_sql_file $WORK_DIR/db1.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
 
+	export GO_FAILPOINTS="github.com/pingcap/dm/loader/loaderCPUpdateOffsetError=return(1)"
```

Comment:
I remember failpoint could use `return()` to trigger an inject, not sure if there's a more simple way